sitemedia/scss/components/_transcription.scss

```diff
@@ -563,18 +563,36 @@
                 }
             }
         }
-        img {
+        .deep-zoom-container {
+            position: relative;
             width: 100%;
-            margin: 0 auto;
-            &.hidden {
-                transition: opacity 300ms ease, visibility 0s ease 300ms;
-                opacity: 0;
-                visibility: hidden;
-            }
-            &.visible {
-                transition: opacity 300ms ease, visibility 0s ease 0ms;
-                visibility: visible;
-                opacity: 1;
+            text-align: center;
+            img {
+                width: auto;
```

Comment:
Why did you set image width to `auto` ? I think on mobile it should still be `100%`, otherwise we're losing significant resolution on the static images (at least on the long ones). I tested locally and it looks like that still works properly (might need some kind of style tweak for non-mobile to make sure the image is centered within the panel, or maybe I just missed something in my quick check.)